Assets/Scripts/UI/DialogBox/Settings/DialogBoxSettings.cs

```diff
@@ -78,7 +78,7 @@ public void OnClickSave()
         this.CloseDialog();
         WorldController.Instance.spawnInventoryController.SetUIVisibility(developerModeToggle.isOn);
         LocalizationTable.SetLocalization(languageDropdown.value);
-        //FurnitureBuildMenu.instance.RebuildMenuButtons(developerModeToggle.isOn);
+        ///FurnitureBuildMenu.instance.RebuildMenuButtons(developerModeToggle.isOn);
```

Comment:
Yes you should replace your extra / by a extra space. Comment should start by a space, don't ask me why, stylecop rule.